src/config.rs

```diff
@@ -135,6 +135,11 @@ pub struct Config {
     /// Allow dubious host names.
     pub allow_dubious_hosts: bool,
 
+    /// Should we wipe store and cache before starting?
```

Comment:
The config argument documentation refers to "store", not "cache", the code changes seem only to affect "caches" and this comment refers to both "store" and "cache". Would it be helpful to be more consistent in the naming?